examples/hello-world/hello-cyclic/README.md

```diff
@@ -25,42 +26,39 @@ Prepare the data first:
 bash ./prepare_data.sh
 ```
 
-Use nvflare simulator to run the hello-examples:
+Run the script using the job API to create the job and run it with the simulator:
 
-```bash
-nvflare simulator -w /tmp/nvflare/ -n 2 -t 2 ./jobs/hello-cyclic
+```
+python3 cyclic_script_executor_hello-cyclic.py
 ```
 
 ### 3. Access the logs and results
 
-You can find the running logs and results inside the simulator's workspace/simulate_job
+You can find the running logs and results inside the simulator's workspace:
 
 ```bash
-$ ls /tmp/nvflare/simulate_job/
-app_server  app_site-1  app_site-2  log.txt
-
+$ ls /tmp/nvflare/jobs/workdir
 ```
 
 ### 4. Notes on running with GPUs
 
-For running with GPUs, we recommend using
+For running with GPUs, we recommend using the
 [NVIDIA TensorFlow docker](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tensorflow)
 
-If you choose to run the example using GPUs, it is important to note that,
-by default, TensorFlow will attempt to allocate all available GPU memory at the start.
+If you choose to run the example using GPUs, it is important to note that by default, TensorFlow will attempt to allocate all available GPU memory at the start.
 In scenarios where multiple clients are involved, you have a couple of options to address this.
 
 One approach is to include specific flags to prevent TensorFlow from allocating all GPU memory.
-For instance:
+For instance if you are running the job with the simulator after exporting it with `job.export_job("/tmp/nvflare/jobs/job_config")` uncommented at the end of script in this directory:
 
 ```bash
-TF_FORCE_GPU_ALLOW_GROWTH=true nvflare simulator -w /tmp/nvflare/ -n 2 -t 2 ./jobs/hello-cyclic
+TF_FORCE_GPU_ALLOW_GROWTH=true nvflare simulator -w /tmp/nvflare/ -n 2 -t 2 /tmp/nvflare/jobs/job_config/hello-tf_cyclic
```

Comment:
Do users have to export?
What if they just specify this flag when running the "python3 cyclic_script-executor-hello-cyclic.py"?

Reply:
Instead of asking users to use job api to export then run this nvflare simulator command,

Can it be just one liner? `TF_FORCE_GPU_ALLOW_GROWTH=true  python3 cyclic_script-executor-hello-cyclic.py`

Reply:
Thanks, updated it to simplify it. I'm not sure about the section for so removed it for now:

If you possess more GPUs than clients,
an alternative strategy is to run one client on each GPU.

Reply:
yeah that statement is not 100% solving this issue, we can remove that